linera-views/src/backends/dual.rs

```diff
@@ -352,7 +352,7 @@ where
 #[derive(Error, Debug)]
 pub enum DualStoreError<E1, E2> {
     /// Serialization error with BCS.
-    #[error("BCS error: {0}")]
+    #[error(transparent)]
     Bcs(#[from] bcs::Error),
```

Comment:
```suggestion
    BcsError(#[from] bcs::Error),
```

Reply:
Ok, I put the `BcsError`, `JsonError`, and `IoError`. Those are clear.
So, merge the PR, and then go forward from there.